### ISSUE TITLE
Modloading Crash: Cannot set boolean property of LeadDoorBlock

### ISSUE THREAD

**Issue by DWalz** [MehVahdJukaar/Supplementaries#197]
**Mod Version:** I'm using the newest state of the master branch: b75e3101945f08ab815a346856116d6c403beee4
**Forge Version:** Newest release (forge-1.18.1-39.0.66)
**Exception:** java.lang.IllegalArgumentException: Cannot set property BooleanProperty{name=powered, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{null}

I had a quick look at the 1.2.0 commit: 7f7c832d8748fa6fafcd51dd103d6513b5d21180. It seems that with the addition of `LeadDoorBlock` the `createBlockStateDefinition` function misses `BooleanProperty Block.POWERED`:

```java
protected void createBlockStateDefinition(StateDefinition.Builder<Block, BlockState> pBuilder) {
    pBuilder.add(HALF, FACING, OPEN, HINGE, OPENING_PROGRESS);
}
```

Which cannot be set later.

Should I create a pull request for this issue?

**Comment by MehVahdJukaar** [comment]
oh yes that block isnt finished a and I'm currently working on it. I did leave it there becasue I was too lazy to remove it and I didnt think anybody would have used it. Unfortunately I didnt consider that it was in the creative tab so yeah... I'll have it ready next update tho. No need for pr since I've already got that sorted in my dev version

**Comment by MehVahdJukaar** [comment]
fixed in 1.2.1
